src/Starlight/Controls.hs

```diff
@@ -25,23 +25,24 @@ controls
 controls = do
   input <- get
   let actions = catMaybes (map (runRelation input) controlRelations)
-  Set.fromList <$> for actions (\ (input, action) ->
-    action <$ modify (\\ input))
+  Set.unions <$> for actions (\ (input, actions) ->
+    actions <$ modify (\\ input))
 
 -- FIXME: make this user-configurable
-controlRelations :: [Relation Input (Input, Action)]
+controlRelations :: [Relation Input (Input, Set.Set Action)]
```

Comment:
It occurred to me that the AI can perform multiple actions in a frame, player input should be able to chord them too.